string switchGripperName
---
bool is_success
string info
